mlrun/runtimes/sparkjob.py

```diff
@@ -332,6 +340,10 @@ def _get_driver(self, name, namespace=None):
         _ = self._get_k8s()
         return list(pods.items())[0]
 
+    @property
+    def is_deployed(self):
+        return True
+
```

Comment:
I want to still allow to build a new image using MLRun, let's change it to:
```suggestion
    @property
    def is_deployed(self):
        if not self.spec.build.source and not self.spec.build.commands:
            return True
        return super().is_deployed

```